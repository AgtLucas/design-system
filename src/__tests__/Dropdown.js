import React from 'react'
import renderer from 'react-test-renderer'
import { Dropdown } from '..'

describe('Dropdown', () => {
  test('renders', () => {
    const json = renderer.create(<Dropdown>Dropdown</Dropdown>).toJSON()
    expect(json).toMatchSnapshot()
  })
})
