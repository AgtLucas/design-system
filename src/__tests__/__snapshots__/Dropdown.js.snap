// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown Dropdown renders 1`] = `
.c1 {
  text-align: center;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="c0 c1"
  direction="column"
  width={undefined}
>
  Dropdown
</div>
`;
